dependencies.yaml

```diff
@@ -245,32 +245,33 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - GPUtil>=1.4.0
-          - imagecodecs>=2021.6.8
-          - openslide-python>=1.1.2
           - psutil>=5.8.0
           - pytest-cov>=2.12.1
           - pytest-lazy-fixture>=0.6.3
           - pytest-xdist
           - pytest>=6.2.4
           - tifffile>=2022.7.28
+          - pooch>=1.6.0  # needed to download scikit-image sample data
       - output_types: [conda]
         packages:
+          - imagecodecs>=2021.6.8
+          - matplotlib-base
+          - openslide-python>=1.1.2
           - pip
           - pip:
             - opencv-python-headless>=4.6
+      - output_types: [requirements]
+        packages:
+          - imagecodecs>=2021.6.8
+          - openslide-python>=1.1.2
+      # temporarily remove imagecodecs / openslide-python from wheel tests
```

Comment:
Maybe this code comment affects the review comments I left above. I don't see manylinux ARM builds of imagecodecs. I'll leave it to you to figure out the best course of action here.

Reply:
Yeah, I need to open an issue to fix running tests when `imagecodecs` is installed (there is a single test case that fails on CI, but I couldn't reproduce using a wheel built on my local system). There was also an issue on arm64 where it tries to compile from source, but fails (only x86_64 wheels were available)

I went ahead and merged the requirements and pyproject sections and just comment it out in for now.

Reply:
opened issue https://github.com/rapidsai/cucim/issues/626 to track